Create em.json

immanwell opened the pull request:
Personal site and email for em.is-a.dev

<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

Hello is-a.dev team,

My server at 129.213.127.217 (the IP address in my A record) is configured to drop direct IP connections for security.

To prove the server is live and the content for my site is ready, I have hosted a live preview at the following URL, which points to the same server:

[http://dev.emmanuel.mba](https://www.google.com/search?q=http://dev.emmanuel.mba)

The Nginx configuration for em.is-a.dev is in place and ready to go as soon as the domain is active.

Thank you for your consideration!

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I have **read** and **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->
![CleanShot 2025-06-30 at 05 15 47@2x](https://github.com/user-attachments/assets/695d616b-5563-4186-b7cb-57a77e8704d2)

immanwell committed:
Create em.json

Personal site and email for em.is-a.dev

STICKnoLOGIC commented on domains/em.json:
```diff
@@ -0,0 +1,19 @@
+      "10 mail.sitemgt.io"
+    ]
+  },
+  "subdomains": {
```
there is no ```"subdomain"```, you need to create another json file e.g. subdomain.yourdomain.json.
to know more, #rtfd https://docs.is-a.dev/domain-structure/

Best Regards,
STICKnoLOGIC

21Z reviewed (dismissed):
what sticknologic said

immanwell committed:
Update em.json

Thank you! To simplify, I've removed the request for the www subdomain and corrected the file.

immanwell committed:
Update em.json

Thank you! To simplify, I've removed the request for the www subdomain and corrected the file.

immanwell committed:
Merge branch 'main' into patch-2
